Source/SwiftLintFramework/Rules/FirebaseConfigDefaultsRule.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  FirebaseConfigDefaultsRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigDefaultsRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_defaults",
+        name: "Firebase Config Defaults",
+        description: "Firebase Config defaults should be set.",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+            "        (status, error) -> Void in \n" +
+            "    }\n" +
+            "    remoteConfig.setDefaults(fromPlist: \"RemoteConfigDefaults\")\n" +
+            "  }\n" +
+            "}"
+        ],
+        triggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+                "  override func viewDidLoad() {\n" +
+                "    super.viewDidLoad() \n" +
+                "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                "        (status, error) -> Void in \n" +
+                "    }\n" +
+                "  }\n" +
+            "}"
+        ]
+    )
+
+    public func validate(file: File, kind: SwiftExpressionKind,
+                         dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        guard SwiftExpressionKind.call == kind else {
+            return []
+        }
+
+        guard dictionary.name!.hasSuffix(".fetch") else {
+            return []
+        }
+
+        guard dictionary.substructure[0].name == "withExpirationDuration" else {
```

Comment:
Same issue trying to access the first element of the array (it might not have a first element).